source/tutorial.rst

```diff
@@ -461,6 +461,62 @@ Swaps the elements at ``index1`` and ``index2`` in the array. The optional ``lon
 
 Returns the fixed size of the array.
 
+Tree<T>
+###############
+
+The tree type encapsulates an underlying Node<T> object. The distinction between the Node and Tree types exist to allow you to be able to specify exactly which nodes in a subtree should be animated.
```

Comment:
Really minor grammar thing - don't think we need both "allow" and "be able to" here - just to make the sentence more straight forward.